packages/networks/src/networks/sepolia.ts

```diff
@@ -84,6 +84,7 @@ export const sepolia: NetworkConfig = {
     networkName: 'sepolia',
     studioName: 'unlock-protocol-sepolia',
   },
+  swapPurchaser: '0x580A4a5a9612371e3832c4559026B9Ee3b23Cc11',
```

Comment:
Deplpyed on Sepolia but can't test because we're getting 404 on 
```
failed to resolve /ipns/api.uniswap.org/v1/pools/v3/sepolia.json: no link named "sepolia.json" under QmaimGCW33MN3uCAZYQmckXLCFQCgt82bypYt4tWqpiU32
```